Mage.Sets/src/mage/cards/s/SeasonOfGathering.java

```diff
@@ -0,0 +1,162 @@
+package mage.cards.s;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+import mage.abilities.Ability;
+import mage.abilities.Mode;
+import mage.abilities.dynamicvalue.common.ControlledCreaturesGreatestPower;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.*;
+import mage.abilities.effects.common.continuous.GainAbilityTargetEffect;
+import mage.abilities.keyword.TrampleAbility;
+import mage.abilities.keyword.VigilanceAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.choices.Choice;
+import mage.choices.ChoiceImpl;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.counters.CounterType;
+import mage.filter.FilterPermanent;
+import mage.filter.StaticFilters;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.common.TargetControlledCreaturePermanent;
+import mage.target.targetpointer.FixedTarget;
+
+/**
+ *
+ * @author jimga150
+ */
+public final class SeasonOfGathering extends CardImpl {
+
+    public SeasonOfGathering(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.SORCERY}, "{4}{G}{G}");
+
+        // Choose up to five {P} worth of modes. You may choose the same mode more than once.
+        this.getSpellAbility().getModes().setMaxPawPrints(5);
+        this.getSpellAbility().getModes().setMinModes(0);
+        this.getSpellAbility().getModes().setMaxModes(5);
+        this.getSpellAbility().getModes().setMayChooseSameModeMoreThanOnce(true);
+
+        // {P} -- Put a +1/+1 counter on a creature you control. It gains vigilance and trample until end of turn.
+        this.getSpellAbility().addEffect(new SeasonOfGatheringCounterEffect());
+        this.spellAbility.getModes().getMode().setPawPrintValue(1);
+
+        // {P}{P} -- Choose artifact or enchantment. Destroy all permanents of the chosen type.
+        Mode mode2 = new Mode(new SeasonOfGatheringRemovalEffect());
+        this.getSpellAbility().addMode(mode2.withPawPrintValue(2));
+
+        // {P}{P}{P} -- Draw cards equal to the greatest power among creatures you control.
```

Comment:
Why paw logic done by mode’s min/max style instead mode cost? Is it a good design here for it? @theelk801

Reply:
We don't have the CR update yet, but {P} is not a cost.

From [mechanics article](https://magic.wizards.com/en/news/feature/bloomburrow-mechanics)
> As you cast one of these modal spells, you can choose up to five pawprints worth of modes. Pawprints are a new symbol used exclusively for these cards. They aren't a resource like mana or energy. You can't save them or carry them over from one spell to the next.

So it seems to me that this is appropriate design.

Reply:
Then it’s fine